<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1427</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:#DCDCDC;
border-right:1px solid black;
border-left:1px solid black;
border-top:1px solid black;
border-bottom:1px solid black;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}
QPushButton{
    border:none;
    border-bottom:1px solid black;
    font-size:18px;
    font-weight:700;
 }
QPushButton:hover{border-left:4px solid grey;font-weight:700;}</string>
  </property>
  <widget class="QPushButton" name="left_visit">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="realTimeCaptureGroupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>781</width>
     <height>691</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>摄像头实时画面</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="Capturelabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>720</width>
      <height>640</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>摄像头未开启</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Voicevideo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>语音播报</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Voicerecognition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>语音识别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="left_close">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="logGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>系统日志</string>
   </property>
   <widget class="QTextEdit" name="logtextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="left_mini">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Openvideo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Facerecognition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>人脸识别</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>40</y>
     <width>361</width>
     <height>751</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{border:none}</string>
   </property>
   <property name="title">
    <string>验证信息</string>
   </property>
   <widget class="QLabel" name="person1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="person2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="person3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>151</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person1name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>学号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person1stuid">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>英文名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person1english">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>151</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person2name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>学号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person2stuid">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>英文名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person2english">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>151</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person3name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>学号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person3stuid">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{border:none}</string>
       </property>
       <property name="text">
        <string>英文名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="person3english">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border:none}</string>
   </property>
   <property name="title">
    <string>设置</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string>置信度阈值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="confidenceThresholdSlider">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="confidencelabel">
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string>处理图像大小</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="setsizeSlider">
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="setsizelabel">
       <property name="font">
        <font>
         <family>华文中宋</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
